src/index.html

```diff
@@ -1,13 +1,376 @@
+            class="menu__call menu__call--action"
+          >
+          call to order
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main class="main">
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-4">
+              <h2 class="about__title">
+                Miami best
+                <br>
+                Real estate
+
+                <a href="#" class="about__title-link">here</a>
+              </h2>
+            </div>
+            <div class="grid__item grid__item--7-12">
+              <a href="#" class="about__subtitle">
+                Over 50 years experience in love.
+              </a>
+              <p class="about__paragraph">
+                Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+              </p>
+              <p class="about__paragraph">
+                The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.
+                <br>
+                Working directly with developers - so have more useful information about the profitable deals.
+              </p>
+              <p class="about__paragraph about__paragraph--important">
+                We will never be late!
+                <br>
+                Guaranteed secure transaction - 100%
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section" id="gallery">
+        <div class="container">
+          <h2 class="page__section-title">
+            Gallery
+          </h2>
+          <div class="place gallery__place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-8">
+                  <div class="place__photo-container">
+                    <img
+                      class="place__photo"
+                      src="./images/photos/1.jpg"
+                      alt="Place 1"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--desktop-9-12">
+                  <div class="place__photo-container">
+                    <img
+                      class="place__photo"
+                      src="./images/photos/2.jpg"
+                      alt="Place 1"
+                    >
+                  </div>
+                </div>
+              </div>
+
+              <div class="place__info">
+                <a href="https://goo.gl/maps/HWQWNWPpi8tKiWgc6" class="place__info-link">Marina Palms</a>
+                / North Miami Beach, FL 33162
```

Comment:
look again at the instructions. fix everywhere 
https://github.com/mate-academy/layout_miami